jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -313,6 +313,8 @@
             # verify-godeps compares against upstream, but remote/master might be stale
             # if .git was retained between runs. Update it explicitly here.
             git fetch remote master:refs/remotes/remote/master
+            # similarly, verify-munge-docs compares against the latest release branch.
+            git fetch remote release-1.4:remote/release-1.4
```

Comment:
Right now it's hardcoded into mungedocs.go.

It's not a great solution, just the minimal fix. Ideally we'll be doing the git operations ourselves before 1.5, to fix this and the 'ref is not a tree' flakes. Then we ensure we've fetched master and all the release branches.